tasks/buildimages.py

```diff
@@ -82,19 +87,36 @@ def generate_pr_body(
 @task(
     help={
         "file_path": "path of the Gitlab configuration YAML file",
+        "image_type": "The type of image to get the tag for. Used for CI_IMAGE_ variables",
     },
     autoprint=True,
 )
-def get_tag(_, file_path=".gitlab-ci.yml"):
+def get_tag(_, file_path=".gitlab-ci.yml", image_type=None):
     """
     Print the current image tag of the given Gitlab configuration file (default: ".gitlab-ci.yml")
     """
     yaml.SafeLoader.add_constructor(ReferenceTag.yaml_tag, ReferenceTag.from_yaml)
     with open(file_path) as gl:
         gitlab_ci = yaml.safe_load(gl)
-    if "variables" not in gitlab_ci or "DATADOG_AGENT_BUILDIMAGES" not in gitlab_ci["variables"]:
+    if "variables" not in gitlab_ci:
         raise Exit(
-            color_message(f"Impossible to find the version of image in {file_path} configuration file", "red"),
+            f'[{color_message("ERROR", "red")}] - No variables in gitlab configuration file {file_path}',
+            code=1,
+        )
+
+    if image_type is None:
+        return gitlab_ci["variables"].get(
+            "DATADOG_AGENT_BUILDIMAGES",
+            f'{color_message("Not found", "red")}: DATADOG_AGENT_BUILDIMAGES is not defined in the configuration',
+        )
+    else:
+        available_images = set()
+        for key in gitlab_ci["variables"].keys():
+            if key.startswith("CI_IMAGE"):
+                available_images.add(key.removeprefix("CI_IMAGE_").removesuffix("_SUFFIX").casefold())
+                if image_type in key.casefold():
```

Comment:
Why `in` ? Why not just equals ?

Reply:
If I understand correctly if someone used `deb` then they would get the version of the first `deb` image listed, but each image could have a different version so you can't know which one you'll get a priori

Reply:
Yes for this tasks it's not useful